README.md

```diff
@@ -29,35 +27,17 @@ where everything falls into place in the new approach.
 | `{integration}/test_check.py` | `{integration}/tests/*.py`                        |
 | n/a                           | `{integration}/setup.py`                          |
 
-## A note about installing
-
 Now that integrations are cleanly defined as python packages, we will soon be able
 to ship them as Python wheels that will be pip-installable in the Python environment
 embedded into the Datadog Agent. This presents a paradigm change in the way we will
 be delivering standalone integration upgrades, moving away from OS-specific packages
 to idiomatic Python package delivery.
 
-Agent releases starting from version 5.21 bundle the latest wheels for any
-integration, but at the moment you can't upgrade or downgrade between releases.
```

Comment:
single check update feature won't be backported to A5, let's remove any potentially confusing mention